peerpod-ctrl/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM --platform=$TARGETPLATFORM quay.io/confidential-containers/golang-fedora:1.20.8-36 as builder
+FROM --platform=$TARGETPLATFORM quay.io/confidential-containers/golang-fedora:1.20.8-38 as builder
```

Comment:
@snir911 the golang-fedora container update is required for the ssh authentication or you are just using this PR to bump to latest?

Reply:
you got me :P